crypto/LPdir_unix.c

```diff
@@ -121,7 +121,7 @@ const char *LP_find_file(LP_DIR_CTX **ctx, const char *directory)
 
 #ifdef __VMS
     strncpy((*ctx)->previous_entry_name, (*ctx)->entry_name,
-            sizeof((*ctx)->previous_entry_name));
+            sizeof((*ctx)->previous_entry_name) - 1);
```

Comment:
This one was detected by `grep`ping for `strncpy` usage combined with `sizeof`, and I cannot test compilation on VMS.

Reply:
If accepted, this change would also apply to `1.1.1`, the others are `master`-specific.

I would open a separate PR for this.

Reply:
This change actually looks wrong, the old code was correct. Both previous_entry_name and entry_name are the same size.

Reply:
ok, reverting!

Thanks for the feedback @kroeckx I was indeed quite misguided working on this the way I approached it!

Reply:
but you did not push the revert?

Reply:
> but you did not push the revert?
> but you did not push the revert?

Yeah, I got lost while trying to figure out why I was seeing other failures, before realizing that those are happening for me even before these changes!

Pushed now a fixup commit, changes to this file are dropped.